<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFlowGraphEditModuleDlg</class>
 <widget class="QDialog" name="CFlowGraphEditModuleDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>336</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="FG_MODULE_GROUP_1">
     <property name="title">
      <string>Inputs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item row="0" column="0" rowspan="4">
       <widget class="QListWidget" name="FG_MODULE_INPUT_LIST"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="ADD_MODULE_INPUT">
        <property name="text">
         <string>New Input</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="DELETE_MODULE_INPUT">
        <property name="text">
         <string>Delete Input</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="EDIT_MODULE_INPUT">
        <property name="text">
         <string>Edit Input</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="FG_MODULE_GROUP_2">
     <property name="title">
      <string>Outputs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item row="0" column="0" rowspan="4">
       <widget class="QListWidget" name="FG_MODULE_OUTPUT_LIST"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="ADD_MODULE_OUTPUT">
        <property name="text">
         <string>New Output</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="DELETE_MODULE_OUTPUT">
        <property name="text">
         <string>Delete Output</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="EDIT_MODULE_OUTPUT">
        <property name="text">
         <string>Edit Output</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>32</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CFlowGraphEditModuleDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CFlowGraphEditModuleDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
